admin-console/src/main/java/com/agiletec/apsadmin/category/CategoryAction.java

```diff
@@ -260,31 +271,29 @@ protected void extractReferencingObjects(String categoryCode) {
 			_logger.error("Error extracting referenced objects by category {}", categoryCode, t);
 		}
 	}
-	
+
 	public String save() {
 		String parentCategoryCode;
 		try {
-			if(this.getParentCategoryCode().contains(",") && !StringUtils.endsWith(this.getParentCategoryCode(), ",")) {
+			if (this.getParentCategoryCode().contains(",") && !StringUtils.endsWith(this.getParentCategoryCode(), ",")) {
 				parentCategoryCode = StringUtils.trim(this.getParentCategoryCode().split(",")[1]);
 			} else {
 				parentCategoryCode = this.getParentCategoryCode();
 			}

```

Comment:
why not a ternary?
which method here can actually throw on exception? why inside the try - catch?